components/coda/actions/copy-doc/copy-doc.mjs

```diff
@@ -0,0 +1,41 @@
+import coda from "../../coda.app.mjs";
+
+export default {
+  key: "coda_copy-doc",
+  name: "Copy Doc",
+  description: "Creates a copy of the specified Coda doc",
+  version: "0.0.17",
+  type: "action",
+  props: {
+    coda,
+    sourceDoc: {
+      propDefinition: [
+        coda,
+        "sourceDoc",
+      ],
+      description: "The doc ID from which to create a copy.",
```

Comment:
You can remove the "." from the end of this, since it's not a complete sentence

Reply:
This description is also virtually the same as the one in the prop definition. Maybe remove it here?

Reply:
Issue addressed